bootstrap.sh

```diff
@@ -63,11 +70,11 @@ export PYTHONPATH=$(pwd)/external:$(pwd)/external/usr/lib/python$pyver/site-pack
 CMD $python -m pip install --no-cache-dir -q --upgrade pip --target=external/
 CMD $python -m pip install --use-feature=2020-resolver --no-cache-dir -q -r requirements.txt --target=external/ --upgrade
 
-if [ x"$1" == x"+docs" -o x"$2" == x"+docs" ]; then
+if [ "$MAKEDOCS" == "true" ]; then
     CMD $python -m pip install --use-feature=2020-resolver --no-cache-dir -q -r docs/requirements.txt --target=external/ --upgrade
     make -C docs PYTHON=$python
 fi
 
-if [ x"$1" == x"+pygelf" -o x"$2" == x"+pygelf" ]; then
+if [ "$PYGELF" == "true" ]; then
     CMD $python -m pip install --use-feature=2020-resolver --no-cache-dir -q pygelf --target=external/ --upgrade
```

Comment:
We should pin that version, too, but I don't want to pin it here. What if we introduced it in the `requirements.txt` like that?

```python
#.pygelf pygelf==0.3.6
```

So then you create a new temporary `requirements.txt` by uncommenting this line with sed and use that for installing everything.